qbx_core.sql

```diff
@@ -1,3 +1,13 @@
+CREATE TABLE IF NOT EXISTS `users` (
+  `userId` int UNSIGNED NOT NULL AUTO_INCREMENT,
+  `username` varchar(50) DEFAULT NULL,
```

Comment:
Should we store this info? This doesn't seem like a useful field

Reply:
It's there so you can see whos who in the db. Otherwise its just a bunch of numbers and letters. It's also easier to filter the DB by the name if you don't know the license. We have it in the players table with all their characters as well.